### PR TITLE
docs: Add mastodon link

### PR DESCRIPTION
**Pull request by amareshsm**
#### Prerequisites checklist

- [x] I have read the [contributing guidelines](https://github.com/eslint/eslint/blob/HEAD/CONTRIBUTING.md).

#### What is the purpose of this pull request? (put an "X" next to an item)

[ ] Documentation update
[ ] Bug fix ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/bug-report.md))
[ ] New rule ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/rule-proposal.md))
[ ] Changes an existing rule ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/rule-change-proposal.md))
[ ] Add autofix to a rule
[ ] Add a CLI option
[ ] Add something to the core
[x] Other, please explain:
Added Mastodon link along with our other social buttons.

#### What changes did you make? (Give an overview)
Added Mastodon link along with our other social buttons.
image

### CODE REVIEW

File: docs/src/_includes/components/social-icons.html [hunk 1]
```diff
@@ -52,6 +52,13 @@
 
             </a>
         </li>
-
+        <li>
+            <a href="{{ links.mastodon }}" target="_blank">
```
[the remark this answers is not in the document]
Comment (harish-sethuraman) [reply docs/src/_includes/components/social-icons.html:56]: Should we add referrer? 👀

[the remark this answers is not in the document]
Comment (amareshsm) [reply docs/src/_includes/components/social-icons.html]: yes, we can add.

**Review by harish-sethuraman — APPROVED** [review]
LGTM

**Comment by chenxsan** [comment]
If we're going to wrap the icons, maybe we could update the `padding-top` and `padding-bottom` for `.eslint-social-icons ul li a` as well?

Currently the vertical space between two icons is larger than horizontal one:



Which makes them look like two separate parts instead of a cohesive one when wrapped:

**Review by nzakas — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**